tests/wpt/web-platform-tests/cssom-view/offsetPropertiesInline.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>cssom-view - offsetParent, offsetTop, offsetLeft, offsetWidth, and offsetHeight on inline elements</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  #real-offset-parent {
+    font: 10px/1 Ahem;
+    margin: 0;
+    border: none;
+    padding: 0;
+  }
+  #real-offset-parent, #decoy-offset-parent {
+    position: relative;
+  }
+</style>
+<body>
+<div id="real-offset-parent">
+  <span id="inline-1">ABC</span>
+  <span id="inline-2">ABC<br />ABC</span>
+  <span id="inline-3">ABC</span>
+</div>
+<div id="decoy-offset-parent"></div>
```

Comment:
Is `decoy-offset-parent` here for some reason? Seems unused. If so, add a comment about it.

Also, don't put this test in the `web-platform-tests` directory directly (see #12326). Put it under `tests/mozilla/css`.